### PR TITLE
Fix more linter errors in examples

### PR DESCRIPTION
**Pull request by looeee**
Fix linter errors in webgl deferred, webgl2, webaudio, css3d, css2d, raytracing, software, and svg examples.

### CODE REVIEW

File: examples/css3d_molecules.html [hunk 1]
```diff
@@ -206,9 +206,21 @@
 				var b_b = document.getElementById( "b_b" );
 				var b_ab = document.getElementById( "b_ab" );
 
-				b_a.addEventListener( 'click', function() { visualizationType = 0; showAtoms() } );
-				b_b.addEventListener( 'click', function() { visualizationType = 1; showBonds() } );
-				b_ab.addEventListener( 'click', function() { visualizationType = 2; showAtomsBonds() } );
+				b_a.addEventListener( 'click', function () {
+
+					visualizationType = 0; showAtoms();
```
Comment (gero3) [inline examples/css3d_molecules.html]: 2 statements on 1 line??

File: examples/css3d_molecules.html [hunk 2]
```diff
@@ -206,9 +206,21 @@
...
+
+				} );
+				b_b.addEventListener( 'click', function () {
+
+					visualizationType = 1; showBonds();
```
Comment (gero3) [inline examples/css3d_molecules.html]: 2 statements on 1 line??

File: examples/css3d_molecules.html [hunk 3]
```diff
@@ -206,9 +206,21 @@
...
+
+				} );
+				b_ab.addEventListener( 'click', function () {
+
+					visualizationType = 2; showAtomsBonds();
```
Comment (gero3) [inline examples/css3d_molecules.html]: 2 statements on 1 line??

File: examples/raytracing_sandbox.html [hunk 4]
```diff
@@ -283,25 +283,12 @@
...
 				}
 
-				info.innerHTML = '<a href="http://threejs.org" target="_blank" rel="noopener">three.js<a/> - raytracing renderer (using '  + WORKERS + ' <button onclick="updateWorkers(-1)">-</button><button onclick="updateWorkers(1)">+</button> web workers)' +
+				info.innerHTML = '<a href="http://threejs.org" target="_blank" rel="noopener">three.js<a/> - raytracing renderer (using ' + WORKERS + ' <button onclick="updateWorkers(-1)">-</button><button onclick="updateWorkers(1)">+</button> web workers)' +
 				'<br/><button onclick="rearrange()">Rearrange</button><button onclick="render()">Render</button>';
 
 
 			}
 
-			function rearrange() {
```
Comment (gero3) [inline examples/raytracing_sandbox.html]: This function is called in html on [line 287](https://github.com/mrdoob/three.js/blob/4ebbdc8f64fc7f2e9f8bd2c928136ce9b0f63e94/examples/raytracing_sandbox.html#L287)

**Review by gero3 — COMMENTED** [review]
It would be best to change these.

File: examples/css3d_molecules.html (hunk 1 above)
Comment (looeee) [reply examples/css3d_molecules.html]: Hmm, looks like the auto eslint error fixer is missing these.

File: examples/raytracing_sandbox.html (hunk 4 above)
Comment (looeee) [reply examples/raytracing_sandbox.html]: Oh, good catch!

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**